src/hypothesis/internal/conjecture/engine.py

```diff
@@ -1297,9 +1300,8 @@ def greedy_shrink(self):
         # and often actively harmful when run on examples which are "close to
```

Comment:
Do you now want to delete the reference to delta-debugging on the line above?

Reply:
I do, thank you! There's actually a bunch of delta debugging related stuff I should have deleted.